[2023-06-09 15:27:26,967] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: sftp_raw_data_full_load_pipeline.load_sftp_datadata_18.csv manual__2023-06-09T15:20:24.730543+00:00 [queued]>
[2023-06-09 15:27:27,061] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: sftp_raw_data_full_load_pipeline.load_sftp_datadata_18.csv manual__2023-06-09T15:20:24.730543+00:00 [queued]>
[2023-06-09 15:27:27,064] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2023-06-09 15:27:27,066] {taskinstance.py:1242} INFO - Starting attempt 2 of 2
[2023-06-09 15:27:27,069] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2023-06-09 15:27:27,184] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): load_sftp_datadata_18.csv> on 2023-06-09 15:20:24.730543+00:00
[2023-06-09 15:27:27,198] {standard_task_runner.py:52} INFO - Started process 4178 to run task
[2023-06-09 15:27:27,220] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'sftp_raw_data_full_load_pipeline', 'load_sftp_datadata_18.csv', 'manual__2023-06-09T15:20:24.730543+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/sftp_raw_data_full_load_pipeline.py', '--cfg-path', '/tmp/tmpy713518h', '--error-file', '/tmp/tmpw3dmooz5']
[2023-06-09 15:27:27,232] {standard_task_runner.py:77} INFO - Job 91: Subtask load_sftp_datadata_18.csv
[2023-06-09 15:27:27,367] {logging_mixin.py:109} INFO - Running <TaskInstance: sftp_raw_data_full_load_pipeline.load_sftp_datadata_18.csv manual__2023-06-09T15:20:24.730543+00:00 [running]> on host 8cb5811d1998
[2023-06-09 15:27:27,508] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=Hassan@gmail.com
AIRFLOW_CTX_DAG_OWNER=Hassan
AIRFLOW_CTX_DAG_ID=sftp_raw_data_full_load_pipeline
AIRFLOW_CTX_TASK_ID=load_sftp_datadata_18.csv
AIRFLOW_CTX_EXECUTION_DATE=2023-06-09T15:20:24.730543+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-09T15:20:24.730543+00:00
[2023-06-09 15:27:27,534] {logging_mixin.py:109} INFO - error while inserting to table: Invalid file path or buffer object type: <class 'NoneType'>
[2023-06-09 15:27:27,594] {local_task_job.py:154} INFO - Task exited with return code 1
[2023-06-09 15:27:27,659] {taskinstance.py:1280} INFO - Marking task as FAILED. dag_id=sftp_raw_data_full_load_pipeline, task_id=load_sftp_datadata_18.csv, execution_date=20230609T152024, start_date=20230609T152726, end_date=20230609T152727
[2023-06-09 15:27:27,775] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
